languages/csharp/exercises/concept/namespaces/.docs/introduction.md

```diff
@@ -0,0 +1,34 @@
+Namespaces are a way to avoid name clashes and are generally present in all but the most trivial code base.
```

Comment:
Aren't namespaces primarily a way to group related code? The [official docs](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/namespaces/) mention two goals:

- First, .NET uses namespaces to organize its many classes
- Second, declaring your own namespaces can help you control the scope of class and method names in larger programming projects.

It might also be worthwhile to mention that the base class library (or whatever its name is) also heavily uses namespaces to organize its functionality. Although that could also be postponed to the `after.md` file.

Reply:
Modified _introduction.md_ and _after.md_.